k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -158,7 +158,7 @@
             "com.fsck.k9.activity.MessageCompose.quotedTextFormat";
     private static final String STATE_KEY_NUM_ATTACHMENTS_LOADING = "numAttachmentsLoading";
     private static final String STATE_KEY_WAITING_FOR_ATTACHMENTS = "waitingForAttachments";
-
```

Comment:
Please restore the empty line after the group of `STATE_*` constants. This visual separation makes the code easier to read.